src/pages/SetPasswordPage.js

```diff
@@ -104,15 +91,15 @@ class SetPasswordPage extends Component {
         const accountID = lodashGet(this.props.route.params, 'accountID', '');
         const validateCode = lodashGet(this.props.route.params, 'validateCode', '');
 
-        if (this.props.userSignUp.authToken) {
-            Session.changePasswordAndSignIn(this.props.userSignUp.authToken, this.state.password);
+        if (this.props.credentials.authToken) {
+            Session.changePasswordAndSignIn(this.props.credentials.authToken, this.state.password);
         } else {
             Session.setPassword(this.state.password, validateCode, accountID);
         }
     }
 
     render() {
-        const buttonText = this.props.userSignUp.isValidating ? this.props.translate('setPasswordPage.verifyingAccount') : this.props.translate('setPasswordPage.setPassword');
+        const buttonText = !this.props.account.validated ? this.props.translate('setPasswordPage.verifyingAccount') : this.props.translate('setPasswordPage.setPassword');
```

Comment:
Is it possible to consolidate the remaining usage of `verifyingAccount` to `validatingAccount`? 🤔  It seems like we're using both `validating` and `verifying` interchangeably, with the former appearing to be the more common occurrence.